castervoice/lib/ccr/core/nav.py

```diff
@@ -245,7 +245,7 @@ class Navigation(MergeRule):
             R(Function(textformat.clear_text_format)),
         "peek [<big>] format":
             R(Function(textformat.peek_text_format)),
-        "(<capitalization> <spacing> | <capitalization> | <spacing>) (bow|bowel) <textnv> [brunt]":
+        "(<capitalization> <spacing> | <capitalization> | <spacing>) [(bow|bowel)] <textnv> [brunt]":
```

Comment:
Are people are okay with making bowel optional?

Reply:
I would be fine with that but let's open it up in a separate pull request. That way can be discussed separate from this request.